integrations/cdi/narayana-jta-cdi/src/main/java/io/helidon/integrations/narayana/jta/cdi/DelegatingTransactionSynchronizationRegistry.java

```diff
@@ -0,0 +1,299 @@
+/*
+ * Copyright (c) 2019 Oracle and/or its affiliates. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.helidon.integrations.narayana.jta.cdi;
+
+import java.util.Map; // for javadoc only
+
+import javax.transaction.Status; // for javadoc only
+import javax.transaction.Synchronization;
+import javax.transaction.Transaction; // for javadoc only
+import javax.transaction.TransactionManager; // for javadoc only
+import javax.transaction.TransactionSynchronizationRegistry;
+
+/**
+ * An {@code abstract} {@link TransactionSynchronizationRegistry}
+ * implementation that delegates all method invocations to another
+ * {@link TransactionSynchronizationRegistry}.
+ *
+ * <h2>Design Notes</h2>
+ *
+ * <p>This class is {@code public} for convenience.  It is extended by
+ * other non-{@code public} internal classes.</p>
+ *
+ * @author <a href="mailto:laird.nelson@oracle.com">Laird Nelson</a>
+ *
+ * @see TransactionSynchronizationRegistry
+ */
+public abstract class DelegatingTransactionSynchronizationRegistry implements TransactionSynchronizationRegistry {
+
+  private final TransactionSynchronizationRegistry delegate;
```

Comment:
This file seems to be indented with 2 spaces instead of 4.

Reply:
Done.